scripts/bert/compare_tf_gluon_model.py

```diff
@@ -129,8 +136,10 @@
 #                               Gluon MODEL                                   #
 ###############################################################################
 
+vocabulary = tf_vocab_to_gluon_vocab(tokenizer.vocab)
 bert, vocabulary = nlp.model.get_model(args.gluon_model,
-                                       dataset_name=args.gluon_dataset,
+                                       dataset_name=None,
+                                       vocab=vocabulary,
                                        pretrained=not args.gluon_parameter_file,
```

Comment:
We have some converted TF model hosted on s3 that uses the old vocab. It seems that they won't pass the test due to difference in the vocab idx

Reply:
The tests here are failing due to an old bug where duplicate special tokens (eg. `padding_token == eos_token`) would corrupt the `idx_to_token`. These corruptions are detected by some sanity checks in the updated flexible vocab implementation and tests were thus failing.
Backward compatibility for the Bert models should be preserved.

Reply:
See https://github.com/dmlc/gluon-nlp/pull/749